tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/access/MemberAccessTest.java

```diff
@@ -531,6 +543,10 @@ public void testNestedMemberAccessOnIntersectionTypes() {
         BRunUtil.invoke(result, "testNestedMemberAccessOnIntersectionTypes");
     }
 
+    @Test
+    public void testNestedMemberAccessWithSameTypeUnionExprAsIndex() {
+        BRunUtil.invoke(result, "testUnionTypeExprMemberAccess");
+
```

Comment:
Can't see the closing brace of the method. Hence, fail to compile

Reply:
Seems to be a merge conflict. Fixed.